modules/install/rke2_stop_start_instace.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash  
```

Comment:
run a shell check in this file, it has some important improvements

Reply:
done!